README.md

```diff
@@ -97,10 +99,14 @@ LoadLevel Levels/SampleBase/SampleBase.spawnable
 
 If these cfg files are not present, create them as they will be used to when launching server and client launchers.
 
-A server can be run as follows:
+#### Running the Server
+
+> **Note**: Refer to the O3DE document [Test Multiplayer Games in the O3DE Editor](https://o3de.org/docs/user-guide/gems/reference/multiplayer/multiplayer-gem/test-in-editor/), to set up required console variables (cvar) to support play in editor with servers. Ensure you add ```editorsv_enabled=true``` and ```editorsv_launch=true``` to the appropriate .cfg file or to the Editor launcher path. See the [Console Variable Tutorial]((https://o3de.org/docs/user-guide/engine/cvars/#using-the-cvar)) for more details on setting and using cvars.
```

Comment:
The link to "Test Multiplayer Games in the O3DE Editor" covers these cvars. These cvars are for enabling the server in game-mode... I'd suggest either removing the cvars here or changing this section header to "Running the Server in Editor Game Mode". 
I'm trying to avoid the confusion around 3 different ways of "Running the Server". (1) Running the Server automatically when starting editor game mode, (2) running a stand-alone server by hand and then starting editor game mode, and (3) running a standalone server and standalone gamelauncher (not using the editor)

Reply:
I just fixed this prior to your comment. Now we have an extra section

#### Running the Server in the Editor
Refer to the O3DE document [Test Multiplayer Games in the O3DE Editor](https://o3de.org/docs/user-guide/gems/reference/multiplayer/multiplayer-gem/test-in-editor/), to set up required console variables (cvar) to support play in editor with servers. Ensure you configure ```editorsv_enabled``` and ```editorsv_launch``` as required. See the [Console Variable Tutorial]((https://o3de.org/docs/user-guide/engine/cvars/#using-the-cvar)) for more details on setting and using cvars.